src/Nethermind/Nethermind.Runner/configs/op-sepolia_archive.cfg

```diff
@@ -1,20 +1,23 @@
 {
-    "Init" : {
-        "ChainSpecPath" : "chainspec/op-sepolia.json",
-        "GenesisHash" : "0x102de6ffb001480cc9b8b548fd05c34cd4f46ae4aa91759393db90ea0409887d",
-        "BaseDbPath" : "nethermind_db/op-sepolia-archive",
-        "LogFileName" : "op-sepolia-archive.logs.txt",
-        "DisableGcOnNewPayload": false
-    },
-    "Sync": {
-      "NetworkingEnabled": false
-    },
-    "Pruning": {
-       "Mode": "None"
-    },
-    "JsonRpc" : {
-        "Enabled" : true,
-        "Port" : 8545,
-        "EnginePort" : 8551
-    }
+  "Init": {
+    "ChainSpecPath": "chainspec/op-sepolia.json",
+    "GenesisHash": "0x102de6ffb001480cc9b8b548fd05c34cd4f46ae4aa91759393db90ea0409887d",
+    "BaseDbPath": "nethermind_db/op-sepolia-archive",
+    "LogFileName": "op-sepolia-archive.logs.txt",
+    "DisableGcOnNewPayload": false
+  },
+  "TxPool": {
+    "BlobsSupport": "Disabled"
```

Comment:
Is it actually valid that OP stack will have blobs disabled? Will there be some separate release for OP chains to support blobs?